apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -150,6 +150,10 @@ export const getFilter: (
     }
 }
 
+export const getDecimalFilter: (dataIndex: DataIndexType) => FilterType = (dataIndex) => {
```

Comment:
No, it shouldn't be upscaled, since Decimal serach does it under the hood 

From https://v5.keystonejs.com/keystonejs/fields/src/types/decimal/
```
Despite being specified as strings, all comparisons are performed on numerical equivalence. In essence, this means leading and trailing zeros are ignored. Eg. 04.53000 === 4.53.
```